src/content/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure.mdx

```diff
@@ -748,7 +748,7 @@ Standard roles include:
       </td>
 
       <td>
-        Provides ability to configure [authentication domains](/docs/accounts/accounts-billing/new-relic-one-user-management/configure-authentication-domains-sso) for users on the [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models), and manage those users. For how to grant this role, see [Add user management capability](/docs/accounts/accounts-billing/new-relic-one-pricing-users/manage-user-groups-roles-access-grants#add-user-managers).
+        Provides ability to configure [authentication domains](/docs/accounts/accounts-billing/new-relic-one-user-management/configure-authentication-domains-sso) for users on the [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models), and add and manage those users. For how to grant this role, see [Add user management capability](/docs/accounts/accounts-billing/new-relic-one-pricing-users/manage-user-groups-roles-access-grants#add-user-managers).
```

Comment:
Provides "the" ability perhaps?

Reply:
considering it's meant to be succinct and scannable in a table format, I think it's fine.